src/css/prosemirror.scss

```diff
@@ -99,7 +99,7 @@ div.ProseMirror {
 	}
 
 	p .paragraph-content {
-		margin-bottom: 1em;
+		margin-bottom: 0;
```

Comment:
I think we should not generally remove this but only in scope of the callout boxes (warning/error/...)

Reply:
I agree. The `margin-bottom` is also what separates paragraphs from each other to my knowledge, so we have to keep it in general.

Reply:
I agree that the "margin-bottom" is used to separate paragraphs, except that in this case : 

- it creates a systematic, superfluous margin that visually unbalances the content within the block (it creates a kind of ugly extra bottom padding)
- warning blocks need to be compact; we don't put literature in them, but short informative content (so the paragraphs margins a bit useless in this context)

I'd also like to add that I've seen this type of block in the "Polls" app, for example, and they don't have this display issue. So there is a bug to be corrected in this context.

Reply:
Polls : 
![Capture d’écran du 2024-01-25 08-58-52](https://github.com/nextcloud/text/assets/33763786/6b30b08d-ae8c-4baa-bb3e-423d2c0969af)

Reply:
Yes, I think maybe my concern was not clear enough. Your current change also affects paragraphs outside of the warning blocks:

<img width="848" alt="Screenshot 2024-01-25 at 09 10 38" src="https://github.com/nextcloud/text/assets/3404133/55f0bec1-52d8-4d7c-8af4-3bef27ed8917">

For some code pointer we already have a rule for that but it does not seem to apply to the proper element https://github.com/nextcloud/text/blob/e806141296dd11dccfd0f4bb810d9c5b576bed8a/src/nodes/Callout.vue#L91

Reply:
OK, I understand @juliushaertl ; I'll make some changes.